packages/deployment/app/cloudwatchDashboard.yml

```diff
@@ -5,33 +5,41 @@
 # dashboard.
 ##############################################################
 
-## template for creating widgets for Elasticsearch Service alarms
-esTemplateAlarm:
+## template for creating widgets for alarms
+alarmTemplate:
   type: metric
   width: 6
   height: 3
   properties:
-    title: esTemplateAlarm
+    title: alarmTemplate
     annotations:
       alarms:
-      - arn:aws:cloudwatch:{{AWS_REGION}}:{{AWS_ACCOUNT_ID}}:alarm:esTemplateAlarm
+      - arn:aws:cloudwatch:{{AWS_REGION}}:{{AWS_ACCOUNT_ID}}:alarm:alarmTemplate
     view: singleValue
 
-## header widget for ES
-esHeader:
+## header widget for alarms
+alarmHeader:
 - type: text
   width: 24
   height: 1
   properties:
-    markdown: "# Elasticsearch Service"
+    markdown: "## Alarms"
 
-# header widget for ES alarms
-esAlarmHeader:
+## header widget for ECS
+ecsHeader:
 - type: text
   width: 24
   height: 1
   properties:
-    markdown: "## Alarms"
+    markdown: "# Elastic Container Service"
+
+## header widget for ES
+esHeader:
```

Comment:
So this header is always present even if there is no content for the ES dashboard?

Reply:
These are just predefined widgets.  If `es` is not present, the `es` related widgets won't be added to dashboard.  See https://github.com/nasa/cumulus/blob/CUMULUS-803/packages/deployment/lib/kes.js#L199